openshift/helper/kubernetes.py

```diff
@@ -13,6 +14,8 @@
 class KubernetesObjectHelper(BaseObjectHelper):
     @staticmethod
     def client_from_config(config_file, context):
```

Comment:
It seems strange to me to put the logic for handling non-config based client creation in the `client_from_config` function

Reply:
I decided to put it here because the logic for `client_from_config` is in this library. Otherwise, I think I would have had to put the kubernetes specific logic in the kubernetes library and the rest somewhere in `openshift-restclient-python`. This would be inconsistent and I honestly don't think `new_client_from_config` would be a good place for it :(

I think it's fine to have this code here (based on my very limited knowledge of this codebase). Do you think there's a better place for it? Where would you put it? thnx

Reply:
You're probably right, for now best to leave it here. Would you mind adding a TODO over this code branch to look at splitting the non-config logic out of client_from_config. Likely would be a larger refactor, I just want to be reminded whenever I grep the TODOs.

Reply:
Just added them myself, thanks for the patch!